### DISCUSSION TITLE
We've crossed 18 Million Docker Pulls, also a giveaway! 🎉 🐳

### DISCUSSION THREAD

**Discussion by pransh15** [Unleash/unleash#3260] [Announcements]
As y'all can understand from the title, We've crossed 18 Million Docker pulls! :whale:

We are so excited to reach this milestone! This is a testament to the power of our amazing community. :rocket:
Huge thanks to all of you! :heart:

As celebration of reaching this milestone, we are giving away 3 t-shirts for our community from our [Swag Store](https://shop.getunleash.io). How to participate?
1. Star [this repository](github.com/Unleash/unleash)! If you've already done it, jump to the next step. ⏭️
3. Reply to this post with **Done 👕**

We will announce the winners on Friday, 10th of March 2023. Good luck! 💯

**Comment by ghost** [comment]
Done

**Comment by ghost** [comment]
DONE🚀🚀🚀🚀🚀🚀🚀

**Comment by ghost** [comment]
DONE🚀🚀🚀🚀🚀🚀🚀

**Comment by programmersoham** [comment]
Done 👍

**Comment by neelshah2409** [comment]
Done and hoping for best result this time🚀😃✨

**Comment by leoroh** [comment]
Done

**Comment by prathikshetty2002** [comment]
Done

**Comment by testworksau** [comment]
Done 👕

**Comment by ZanMarolt** [comment]
Congrats! Done 👕

**Comment by SolomonJude** [comment]
Done!

**Comment by pransh15** [comment]
Hello Unleashers! 👋

Our community makes it worthwhile to build for, and your contributions mean a lot to us. Thank you so much for participating in our giveaway! ❤️

We had 9 entries in the giveaway, and we are giving away three t-shirts to our community 👕

The moment you were waiting for, is here.... 🥁
Congratulations to our three winners:
1. @SolomonJude 
2. @neelshah2409 
3. @ZanMarolt 

Please DM me(Pranshu - Unleash Team) on the Unleash Slack([https://slack.unleash.run](https://slack.unleash.run)) or email [me](mailto:pranshu@getunleash.io) and I will share the details on obtaining your swag. ❤️

For others, it may not have been your day, but it's not the last you will hear from Unleash. Make sure you follow us on the platform of your choice:
- [GitHub](https://github.com/Unleash)
- [Twitter](https://twitter.com/getunleash)
- [LinkedIn](https://linkedin.com/company/getunleash)

---

In the spirit of total transparency, I wanted to share how we selected those winners. We picked those names at random using wheelofnames.com while documenting the whole process. Feel free to share any feedback to make this process better for you as well as us.

1. I added all the names of our participants to on the wheel
![Screenshot 2023-03-13 at 16 39 55](https://user-images.githubusercontent.com/18158596/224687072-f4abe587-602e-40cf-82fb-f2ead30bccc4.png)

4. Spinned the wheel, and received 3 names as winners! 🎁
![Screenshot 2023-03-13 at 16 42 04](https://user-images.githubusercontent.com/18158596/224686557-093abab3-0e30-49c6-8458-4b33b99ea054.png)
![Screenshot 2023-03-13 at 16 42 28](https://user-images.githubusercontent.com/18158596/224687166-b63b5bd4-52dd-416a-b89e-c6cd569ad99d.png)
![Screenshot 2023-03-13 at 16 42 56](https://user-images.githubusercontent.com/18158596/224687249-ecc23c32-9311-4bdb-9d82-59da2465eddc.png)

Thank you so much for participating! 💟

**Reply by rishiraj88** [reply]
Celebration time!! Congratulations!

**Reply by Fechin** [reply]
Congratulations to all the winners! 🎉 It's great to see such transparency in the selection process using random tools. If you’re looking for another fun way to randomize selections, <a href="https://gospinwheel.com/">gospinwheel.com</a> is also a great option!

**Comment by jonasws** [comment]
@pransh15 Congrats on the great numbers! Can you give any updates about how you're addressing  the upcoming Docker Hub changes? https://blog.alexellis.io/docker-is-deleting-open-source-images/. Will you keep publishing to Docker Hub, or will you move to another hosting provider? Would be great with a heads up if we need to update our pipelines that pull your carefully crafted images :)

**Reply by pransh15** [reply]
Hey @jonasws, I've started a separate thread on the matter for anyone in the future to be able to reference. Thanks for asking! ❤️

**Reply by pransh15** [reply]
Thread Link: https://github.com/orgs/Unleash/discussions/3329
